dockers/agent/core/ngt/Dockerfile

```diff
@@ -14,11 +14,14 @@
 # limitations under the License.
 #
 
+ARG GO_VERSION=latest
 ARG BASE_TAG=latest
 ARG DISTROLESS_IMAGE=gcr.io/distroless/static
 ARG DISTROLESS_IMAGE_TAG=nonroot
 ARG UPX_OPTIONS=-9
 
+FROM golang:${GO_VERSION} AS golang
+
 FROM vdaas/vald-base:${BASE_TAG} AS builder
```

Comment:
do we need copy GOROOT from golang image?
I think vald-base already have GOROOT/{bin,src,misc,pkg}

Reply:
Exactly we need it.

Considering a PR that updates versions/GO_VERSION file, it triggers builds of Docker images.
because GitHub Actions doesn't support dependencies between workflows, we must build vdaas/vald-base and others independently. Then, if agent/core/ngt/Dockerfile does'nt have copying stage of GOROOT, vdaas/vald-agent-ngt built using the former version of Go binaries.

Reply:
I see, thank you for describe it.
LGTM
/format
/rebase
/approve